README.md

```diff
@@ -163,6 +186,57 @@ this with:
 Bugsnag.autoNotify = false;
 ```
 
+###autoBreadcrumbs
+
+By default, we automatically create breadcrumbs for the following types of events:
+
+- **Clicks**
+- **Errors**
+- **Console**
+  - log
+  - warn
+  - error
+- **Navigation**
+  - Page load
+  - DOMContentLoaded
+  - Page hide
+  - Page show
+  - popstate
+  - history.pushState
+  - history.replaceState
+
+
```

Comment:
Maybe a `### Disabling breadcrumbs` header or something would be helpful here, just for easy reading?